cumulusci/cli/cci.py

```diff
@@ -1053,6 +1053,41 @@ def org_list(runtime, plain):
     persistent_table.echo(plain)
 
 
+@org.command(name="prune", help="Removes all expired orgs from the current project")
+@pass_runtime(require_project=False, require_keychain=True)
+def org_prune(runtime):
+    org_configs = {
+        org: runtime.keychain.get_org(org) for org in runtime.keychain.list_orgs()
+    }
+    scratch_configs = getattr(runtime.project_config, "orgs__scratch", {})
+
+    expired_orgs_removed = []
+    org_shapes_skipped = []
+    active_orgs_skipped = []
+    for org, org_config in org_configs.items():
+        if scratch_configs.get(org):
+            org_shapes_skipped.append(org)
```

Comment:
So this is to avoid removing the scratch orgs that are predefined in cumulusci.yml? It might actually be fine to remove them; they should get recreated the next time the keychain is initialized.

If we _do_ need this check, I would probably call it `predefined_scratch_configs` instead of `scratch_configs`, and write the conditional as `if org in scratch_configs:` (you can use `in` to check if a dict contains a key)